client/blocks/edit-gravatar/index.jsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FilePicker from 'components/file-picker';
+import FormLabel from 'components/forms/form-label';
+import {
+	getCurrentUser,
+	isCurrentUserUploadingGravatar
+} from 'state/current-user/selectors';
+import Gravatar from 'components/gravatar';
+import { isOffline } from 'state/application/selectors';
+import { localize } from 'i18n-calypso';
+import Spinner from 'components/spinner';
+import * as OAuthToken from 'lib/oauth-token';
+import { uploadGravatar } from 'state/current-user/actions';
+
+export class EditGravatar extends Component {
+	constructor() {
+		super();
```

Comment:
I like to pass through arguments here, in case we'd like to use props.
```
super( ...arguments );
```

The default derived class constructor also does this too. http://www.2ality.com/2015/02/es6-classes-final.html